actors/market/src/v8/deal.rs

```diff
@@ -123,6 +124,13 @@ impl DealProposal {
     pub fn provider_balance_requirement(&self) -> &TokenAmount {
         &self.provider_collateral
     }
+    pub fn cid(&self) -> anyhow::Result<Cid> {
```

Comment:
Maybe we want to implement `fvm_ipld_encoding::Cbor` for `DealProposal`. The `Cbor` trait also defines `cid()`.

Reply:
It seems to be deprecated now: https://docs.rs/fvm_ipld_encoding/latest/fvm_ipld_encoding/trait.Cbor.html